starter/source/materials/mat/hm_read_mat.F

```diff
@@ -322,7 +322,7 @@ SUBROUTINE HM_READ_MAT(
         MTAG => MLAW_TAG(MAT_NUMBER)
         MATPARAM => MAT_PARAM(MAT_NUMBER)
         MATPARAM%TITLE = ' '
-        MATPARAM%TITLE = TITR(1:LEN_TRIM(TITR)-1)
+        MATPARAM%TITLE = TITR(1:LEN_TRIM(TITR))
```

Comment:
@MaciekWronski, 
The last letter of the title was not added to MATPARAM%TITLE, was it on purpose?